### DISCUSSION TITLE
🔗 CSS Grid UI for Editing Grid Containers (Duplicate of #21175)

### DISCUSSION THREAD

**Discussion by awakekat** [elementor/elementor#24188] [Feature Request]
### Prerequisites

- [ ] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

It is Great that you have incorporated CSS Grid to Elementor, but we need a UI to be able to add measurements to the grid containers. As of November 2023, all we can do is set the grid columns and rows as well as responsive options to change for device sizes. All overrides must be done by writing CSS.

### Describe the solution you'd like

It would be awesome if we had a UI to be able to change the container fr numbers to actually interact with the grid without users having to know the CSS language to make overrides in the custom CSS area. A UI that allowed responsive settings to the grid would be fabulous.

### Describe alternatives you've considered

Tired of writing CSS and media queries for responsive device sizes for CSS Grid.

**Comment by nicholaszein** [comment]
Hello there, @awakekat!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #21175

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards
